index.html

```diff
@@ -75,6 +75,7 @@ <h4>Git and GitHub</h4>
                                     <img src="images/GitHubGit.jpeg" class="img-responsive center-block"
                                         style="height: 150px;" alt="no-image.png">
                                     <ul style="padding-top: 20px;">
+                                        <li class="no-borders-list">Worked with pull requests</li>
```

Comment:
Thanks for the contribution but I don't believe the extra line is required. 

However, from what your trying to add it would be better to instead add Pull Requests in the line **Tagged, branched and merged** to something like **Tagged, branched, pull requests and merges**

Could you please make the required changes, commit and make another pull request?

Thanks.

Reply:
I am making another pull request, in that, I have removed the extra line and added a comment in the tab section.